superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx

```diff
@@ -55,7 +55,12 @@ const RunQueryActionButton = ({
 
   if (shouldShowStopBtn) {
     return (
-      <Button {...commonBtnProps} cta onClick={stopQuery}>
+      <Button
+        {...commonBtnProps}
+        cta
```

Comment:
what's cta here? i think we can remove it

Reply:
it looks like the `cta` prop makes it all caps.